card/lib/card/content.rb

```diff
@@ -59,7 +59,7 @@ def find_chunks chunk_type
       each_chunk.select { |chunk| chunk.is_a?(chunk_type) }
```

Comment:
while you are working on this can you remove the non-existent `msg` variable in line 53?